booking/src/main/java/org/dcsa/conformance/standards/booking/checks/BookingChecks.java

```diff
@@ -136,11 +144,20 @@ public static ActionCheck requestContentChecks(UUID matched, Supplier<CarrierSce
     "All requested Equipments where 'isNonOperatingReefer' is 'false' must have 'activeReeferSettings'",
     ALL_REQ_EQUIP,
     JsonAttribute.ifMatchedThen(
-      IS_ACTIVE_REEFER_SETTINGS_REQUIRED,
+      IS_ISO_EQUIPMENT_CONTAINER_REEFER,
       JsonAttribute.path("activeReeferSettings", JsonAttribute.matchedMustBePresent())
     )
   );
 
+  private static final JsonContentCheck ISO_EQUIPMENT_CODE_AND_NOR_CHECK = JsonAttribute.allIndividualMatchesMustBeValid(
+    "All requested Equipments where ISOEquipmentCode is reefer code must have 'isNonOperatingReefer' flag",
+    ALL_REQ_EQUIP,
+    JsonAttribute.ifMatchedThen(
+      IS_ACTIVE_REEFER_SETTINGS_REQUIRED,
+      JsonAttribute.path("isNonOperatingReefer", JsonAttribute.matchedMustBePresent())
+    )
```

Comment:
If `IS_ACTIVE_REEFER_SETTINGS_REQUIRED` lives up to its name, then `isNonOperatingReefer` would always be false in this case. This strongly suggests that you want a different condition.